src/org/authInfo.ts

```diff
@@ -813,7 +814,16 @@ export class AuthInfo extends AsyncOptionalCreatable<AuthInfo.Options> {
     }
     // If a username with NO oauth options, ensure authorization already exist.
     else if (username && !authOptions && !(await this.stateAggregator.orgs.exists(username))) {
-      throw messages.createError('namedOrgNotFound', [username]);
+      throw SfError.create({
+        name: 'NamedOrgNotFoundError',
+        message: messages.getMessage('namedOrgNotFound', [username]),
+        actions: [
+          `It looks like you mistyped the username or alias. Did you mean "${findSuggestion(username, [
+            ...(await this.stateAggregator.orgs.list()).map((f) => f.split('.json')[0]),
```

Comment:
is splitting the filename better than getting the `username` prop from the contents? Or can we not guarantee that that property will be there? More curious than anything

Reply:
yeah, username is optional in the contents